main.go

```diff
@@ -140,6 +142,17 @@ var (
 	membersCache []User
 )
 
+// blacklistedMemebers returns a list of login names of members which should
+// not be included in pull requests.
+func blacklistedMembers() map[string]bool {
+	ret := map[string]bool{}
```

Comment:
`map[string]struct{}` will fit better. Also `bool{}` is super strange construct :P